packages/react-server-cli/src/commands/start.js

```diff
@@ -83,6 +84,8 @@ const startHtmlServer = (serverRoutes, port, httpsOptions) => {
 			logger.info("Starting HTML server...");
 
 			server.use(compression());
+			server.use(bodyParser.urlencoded({ extended: false }))
+			server.use(bodyParser.json())
```

Comment:
Nice.  We'll probably need to provide some way to pass configuration to bodyParser, eventually, but this will work great for now.